packages/nextjs-mf/src/plugins/container/InvertedContainerPlugin.ts

```diff
@@ -22,55 +11,24 @@ interface InvertedContainerOptions extends ModuleFederationPluginOptions {
   chunkToEmbed: string;
 }
 
-/**
- * The InvertedContainerPlugin is a Webpack plugin that manages the loading of chunks in a federated module.
- * @class
- */
 class InvertedContainerPlugin {
   private options: InvertedContainerOptions;
 
-  /**
-   * Constructs an instance of the InvertedContainerPlugin.
-   * @param {InvertedContainerOptions} options - The configuration options for the plugin.
-   */
   constructor(options: InvertedContainerOptions) {
     this.options = options;
   }
 
-  /**
-   * This method applies the plugin logic and hooks to the Webpack compiler instance.
-   * @param {Compiler} compiler - The Webpack compiler instance.
-   */
   public apply(compiler: Compiler): void {
     new EmbeddedContainerPlugin({
       runtime: this.options.runtime,
       container: this.options.container,
-      chunkToEmbed: this.options.chunkToEmbed,
     }).apply(compiler);
 
-    //@ts-ignore
-    const asyncBoundaryPlugin = new AsyncBoundaryPlugin().apply(compiler);
-    new HoistPseudoEager().apply(compiler);
-    // The following code is commented out for future reference.
-    // asyncBoundaryPlugin.hooks.checkInvalidContext.tap(
-    //   'InvertedContainerPlugin',
-    //   (renderContext, compilation) => {
-    //     return (
-    //       !renderContext ||
-    //       //@ts-ignore
-    //       renderContext?._name ||
-    //       //@ts-ignore
-    //       !renderContext?.debugId ||
-    //       //@ts-ignore
-    //       !compilation.chunkGraph.isEntryModule(renderContext) ||
-    //       //@ts-ignore
-    //       renderContext?.rawRequest?.includes('pages/api') ||
-    //       //@ts-ignore
-    //       renderContext?.layer === 'api'
-    //     );
-    //   },
-    // );
-    // asyncBoundaryPlugin.apply(compiler);
+    new AsyncBoundaryPlugin({
```

Comment:
@2heal1 @zhoushaw in future iteration we should move this inside of Federation. Since ContainerPlugin knows the name of the entry, we can automatically exclude it from encapsulation in asynchronous boundary.

I also think in future we should update federation to that namespace can be promise that resolves to {get,init} | FederationHost - which would enable async wrapping of all cases and allow more flexability in federation when container is promise so "innerResolve" similar to async entrypoint in webpack?